routing/turns_notification_manager.cpp

```diff
@@ -209,7 +213,7 @@ CarDirection NotificationManager::GenerateSecondTurnNotification(vector<TurnItem
   TurnItemDist const & secondTurn = turns[1];
 
   if (firstTurn.m_turnItem.m_index != m_secondTurnNotificationIndex)
-    m_secondTurnNotificationIndex = 0;  // It's a new closest(fisrt) turn.
+    m_secondTurnNotificationIndex = 0;  // It's a new closest(first) turn.
```

Comment:
Space before the parenthesis while we are here.

Reply:
fixed